src/images/badges/it-works-why.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please set the font color to white to work more consistently with other badges.

![image](https://user-images.githubusercontent.com/8878152/92506821-0788fb80-f1d4-11ea-97b0-8e99f0dbb980.png)